spec/features/npq_separation/admin/finance/adjustments_spec.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.feature "Adjustments", type: :feature do
```

Comment:
The spec isn't checking that the page is inaccessible when not logged in - thats crucial to catch any problems from future refactors around auth